### ISSUE TITLE
Creating a widget force closes the app in 0.04-00r1177 nightly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sonuarya/csipsimple#1486]
```
What steps will reproduce the problem?

Install 0.04-00r1177 nightly
Register a sip account
Try to widget for an account switcher

What is the expected output? What do you see instead?
It is supposed to create the widget switcher onto the desktop. Instead it force 
closes. Sometimes, I've had to restart my phone to get the app to respond.

What version of the product are you using? On what operating system?
0.04-00r1177 nightly

Please provide any additional information below.
Basically, trying to create a widget fcs the app.


```

Original issue reported on code.google.com by `oky...@gmail.com` on 10 Jan 2012 at 8:09
